tests/std/tests/P0896R4_ranges_alg_uninitialized_copy/test.cpp

```diff
@@ -275,7 +275,7 @@ struct memcpy_test {
 
 template <test::ProxyRef IsProxy>
 using test_input  = test::range<test::input, int_wrapper, test::Sized::no, test::CanDifference::no, test::Common::no,
-    test::CanCompare::yes, IsProxy>;
+    test::CanCompare::no, IsProxy>;

```

Comment:
This `CanCompare` argument directly controls whether the range's iterators can be compared to each other with `==` and `!=`, and also happens to determine whether iterators are copyable. We want move-only iterators here since they only rewrap rvalues which would fail to compile without the fix to the product code.